transformer_engine/common/include/transformer_engine/fused_attn.h

```diff
@@ -0,0 +1,188 @@
+/*************************************************************************
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+#ifndef TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+#define TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+
+#include "transformer_engine.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*! \brief Compute dot product attention with packed QKV input.
+ *
+ * Computes: BMM1 -> ScaleMaskSoftmax -> Dropout -> BMM2
+ *
+ *  \param[in]     QKV                   The QKV tensor in packed format,
+ *                                       [total_seqs, 3, head, head_dim].
+ *  \param[in]     Bias                  The B tensor.
+ *  \param[in,out] S                     The S tensor, S = Softmax(P), P = Q * K.T.
+ *  \param[out]    O                     The output O tensor.
+ *  \param[out]    Aux_Output_tensors    Auxiliary output tensors when training, e.g. M, ZInv.
```

Comment:
It is called Aus_Output_Tensors (capital T) in the actual function. We should probably use just lowercase letters here to be consistent with the other parameters.

Reply:
Fixed the t -> T. But I'm gonna keep the uppercase A/O/T if no strong objection. Conceptually it is like matrices QKV, S, O, but I understand your concern.

Reply:
OK.